kroxylicious-authorizer-providers/kroxylicious-authorizer-acl/src/main/antlr4/io/kroxylicious/authorizer/provider/acl/parser/AclRules.g4

```diff
@@ -0,0 +1,107 @@
+/**
+ *
+ */
+grammar AclRules;
+
+rule: versionStmt
+    importStmt*
+    denyRule*
+    allowRule*
+    endRule
+    <EOF>
+    ;
+
+versionStmt: 'version' INT SEMI
```

Comment:
I guess 'version' should be shuffled to a token like we've done for the likes of AS,TO,IN.